www/index.php

```diff
@@ -237,68 +244,25 @@
 				</div>
 			</div>
 			
-			<div id="setRootNode" class="fieldgroup" style="">
+			<div id="setRootNode" class="fieldgroup" style="display: none;">
```

Comment:
I like the term "Container" for the divs which just hold other components.  We can id those divs things like "rootNodeToggleContainer", "dynamicNodeToggleContainer"

Buttons can be named after the event which will follow (using a verb), like Sachin suggested.
Checkboxes can be named like "rootNodeCheckbox"
Select boxes and combo boxes can be named like "unitsSelector"
Text inputs/areas can be named "equationInputbox"